layers/+distribution/spacemacs-base/config.el.orig

```diff
@@ -0,0 +1,235 @@
+;; don't let the cursor go into minibuffer prompt
+;; Tip taken from Xah Lee: http://ergoemacs.org/emacs/emacs_stop_cursor_enter_prompt.html
+(setq minibuffer-prompt-properties
+      '(read-only t point-entered minibuffer-avoid-prompt face minibuffer-prompt))
+;; Emacs 24.4 new features
+(unless (version< emacs-version "24.4")
+  (if dotspacemacs-fullscreen-at-startup
+      (spacemacs/toggle-frame-fullscreen)
+    (if dotspacemacs-maximized-at-startup
+        (add-hook 'window-setup-hook 'toggle-frame-maximized))))
+
+;; ---------------------------------------------------------------------------
+;; Session
+;; ---------------------------------------------------------------------------
+
+;; save custom variables in ~/.spacemacs
+(setq custom-file (dotspacemacs/location))
+;; scratch buffer empty
+(setq initial-scratch-message nil)
+;; don't create backup~ files
+(setq backup-by-copying t
+      make-backup-files nil
+      create-lockfiles nil)
+
+;; Auto-save file
+(setq auto-save-default (not (null dotspacemacs-auto-save-file-location)))
+(setq auto-save-list-file-prefix (concat spacemacs-auto-save-directory))
+;; always save TRAMP URLs to cache directory no matter what is the value
+;; of `dotspacemacs-auto-save-file-location'
+(let ((autosave-dir (concat spacemacs-auto-save-directory "dist/")))
+  (setq auto-save-file-name-transforms
+        `(("\\`/[^/]*:\\([^/]*/\\)*\\([^/]*\\)\\'" ,autosave-dir  t)))
+  (unless (or (file-exists-p autosave-dir)
+              (null dotspacemacs-auto-save-file-location))
+    (make-directory autosave-dir t)))
+;; Choose auto-save location
+(case dotspacemacs-auto-save-file-location
+  (cache (let ((autosave-dir (concat spacemacs-auto-save-directory "site/")))
+           (add-to-list 'auto-save-file-name-transforms
+                        `(".*" ,autosave-dir t) 'append)
+           (unless (file-exists-p autosave-dir)
+             (make-directory autosave-dir t))))
+  (original (setq auto-save-visited-file-name t))
+  (_ (setq auto-save-default nil
+           auto-save-list-file-prefix nil)))
+
+;; remove annoying ellipsis when printing sexp in message buffer
+(setq eval-expression-print-length nil
+      eval-expression-print-level nil)
+
+;; cache files
+(setq url-configuration-directory (concat spacemacs-cache-directory "url")
+      eshell-directory-name (concat spacemacs-cache-directory "eshell" )
+      tramp-persistency-file-name (concat spacemacs-cache-directory "tramp"))
+
+;; seems pointless to warn. There's always undo.
+(put 'narrow-to-region 'disabled nil)
+(put 'upcase-region 'disabled nil)
+(put 'downcase-region 'disabled nil)
+(put 'erase-buffer 'disabled nil)
+(put 'scroll-left 'disabled nil)
+(put 'dired-find-alternate-file 'disabled nil)
+;; remove prompt if the file is opened in other clients
+(defun server-remove-kill-buffer-hook ()
+  (remove-hook 'kill-buffer-query-functions 'server-kill-buffer-query-function))
+(add-hook 'server-visit-hook 'server-remove-kill-buffer-hook)
```

Comment:
This doesn't belong here. Work from `develop` and not try to work from `master` and then merge to `develop`.